README.md

```diff
@@ -103,11 +103,10 @@ You should run `./manage.py migrate` frequently, but only use the environment
 variable CCHQ_IS_FRESH_INSTALL during your initial setup.  It is used to skip a
 few tricky migrations that aren't necessary for new installs.
 
-Create a project. The following command will do some basic setup, create a superuser, and create a project. The
-project-name, email, and password given here are specific to your local development environment. Do not include upper-case letters in your project-name. Ignore warnings
-related to Raven for the following three commands.
+Create a superuser for your local environment. (Ignore warnings
+related to Raven for the following three commands.)
 
-    $ ./manage.py bootstrap <project-name> <email> <password>
+    $ ./manage.py make_superuser <email>
```

Comment:
This is nit picky and not important, but `make_superuser` seems kind of ambiguous / hard to differentiate with django's built in `create_superuser`. Could be interesting to just override that command (not sure if that's possible / easy?) or name it something like `create_hq_superuser` just so people don't have to expend mental energy trying to remember whether `make` or `create` is the hq one they want.

Reply:
@dannyroberts  
+1 to this. i found that pretty confusing when helping nitigya but agree that its not important

Reply:
Ah, interesting. I chose "make superuser" in the sense of "make <this user> a superuser", since you can also give it an existing user, to make that user a superuser.

I feel a little funny overriding a standard django command, but there are positives too. Can always still just override it to say "Note: using commcare-hq's custom `make_superuser` command instead" and then call `make_superuser`. That way if someone does run `create_superuser` (1) the right thing happens and (2) the get a clear message that what's happening isn't Django's default but a different thing we decided upon.

I agree this is probably not a huge deal, but it is nice to treat ourselves to these small things working smoothly.

Reply:
Went with something a little different in the same vein: https://github.com/dimagi/commcare-hq/pull/19175. The commands have different arguments so making one just do the other would be kind of clunky.